src/CannedErrorResponses.js

```diff
@@ -0,0 +1,23 @@
+const { Map } = require("immutable");
+
+const cannedErrorResponses = Map({
```

Comment:
Add a reference here on why you have canned error responses for these codes.

Reply:
The representation of (key,value) is not clear for this map. Please add a comment describing the map's purpose.

Reply:
Done.